cmd/monaco/integrationtest/v2/delete_integration_test.go

```diff
@@ -258,7 +258,7 @@ environmentGroups:
 	clientSet := integrationtest.CreateDynatraceClients(t, env)
 
 	// check the setting was deleted
-	integrationtest.AssertSetting(t, context.TODO(), clientSet.SettingsClient, config.SettingsType{SchemaId: "builtin:tags.auto-tagging"}, envName, false, config.Config{
+	integrationtest.AssertSetting(t, t.Context(), clientSet.SettingsClient, config.SettingsType{SchemaId: "builtin:tags.auto-tagging"}, envName, false, config.Config{
```

Comment:
Is it possible to get context from the `t` when it's needed inside the `AssertSetting` function?

NOTE: the unofficial convention is to have context as the first argument.

Reply:
I decided not to use the t.Context inside because we have another place (`assert.go`) where we are adding some values to the context. Should I use a pointer context instead and forward nil if it's not needed?

Reply:
It should be possible to pass `testing.T` here. If not, we need to adapt the testing code.

Reply:
I looked into the code, and the ctx is used for logging - IMO, the client (settings client) should not be aware of logging, but I see why it is currently used that way. 

Let's keep it that way atm

Reply:
In the call, we agreed to investigate the possibility of moving context creation inside the AssertSettings and AssertConfig.

Reply:
Done 👍